opensciencegrid/gwms-factory/etc/osg/image-init.d/30_gwms_start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start factory after upgrade/reconfig is complete
+# Don't delay the rest of the services
+{
+    /usr/sbin/gwms-factory upgrade
+    #/usr/sbin/gwms-factory reconfig
```

Comment:
Let's just delete the commented out line.  Maybe add a comment stating the upgrade also implies a reconfig?